GliaWidgets/Interactor/Interactor.swift

```diff
@@ -92,8 +93,13 @@ class Interactor {
             .filter { $0.0 != nil }
             .forEach {
                 let handler = $0.1
-                DispatchQueue.main.async {
+
```

Comment:
Let's inject DispatchQueue.main.async. We have this under control I suppose via GCD. Also this check can go to GCD.mainQueue live version.

Reply:
fixed in 64bfe8bfa8beb92d3bce127f30d4e957b2aab486